scripts/R/schedule.R

```diff
@@ -0,0 +1,68 @@
+# include init script
```

Comment:
@michaelpcasella when I run schedule.R, I get the following error message in the console:

`Error: Incorrect date value: '' for column 'gameday' at row 1 [1292]`

Seems like the data still populates ok but I think the table can't handle an empty string